#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change --version to report that this is part of Puppet Enterprise,
## DP: and which version of Puppet Enterprise it is part of.

@DPATCH@


From 5d44dea705d23360e9c6e986a5ea325a62aae843 Mon Sep 17 00:00:00 2001
From: Daniel Sauble <djsauble@puppetlabs.com>
Date: Fri, 21 Oct 2011 10:39:58 -0700
Subject: [PATCH 1/2] (#10198) fixes display issue when no nodes in group

* Adds a clearfix so the parent element visually contains all
  floating children.
* Pushes the all_nodes.present conditional up one level so the
  div containing the run failure status will not be displayed
  if there are no nodes in the group
---
 app/views/node_groups/show.html.haml |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/app/views/node_groups/show.html.haml b/app/views/node_groups/show.html.haml
index 698fc05..e69677a 100644
--- a/app/views/node_groups/show.html.haml
+++ b/app/views/node_groups/show.html.haml
@@ -36,8 +36,9 @@
                     = sources.map{|p| link_to(p.name,p)}.join(", ")
       - else
         = describe_no_matches_as 'No child groups'
-  .item
-    - if @node_group.all_nodes.present?
+    .clearfix{:style => 'clear: both'}
+  - if @node_group.all_nodes.present?
+    .item
       .section
         = render 'statuses/run_failure', :nodes => @node_group.all_nodes
   .header
-- 
1.7.4.4

